src/Chainweb/Version.hs

```diff
@@ -777,8 +777,8 @@ twentyChainUpgrade
     -> ChainId
     -> BlockHeight
     -> Bool
-twentyChainUpgrade Mainnet01 _ 745000 = True
-twentyChainUpgrade Development _ 150 = True
+twentyChainUpgrade Mainnet01 _ h = h == 745000
```

Comment:
hmm, is this already decided? Is this safe to put this value here? 

I find it dangerous to put in modification of mainnet01 before we decide on the final fork heights. Otherwise, bad things may happen if the fork gets delayed for whatever reason or we forget about his setting for other reasons.

Reply:
No, this is undecided, and I've raised this. It's just a placeholder

Reply:
There's currently no graph transition configured for `mainnet01`. Shouldn't this here just be `False` in order to be consistent?